scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml

```diff
@@ -232,7 +232,7 @@ scanner:
               Suppressed: DownloadException: No VCS URL provided for 'Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0'.\
               \ Please make sure the published POM file includes the SCM connection,\
               \ see: https://docs.gradle.org/current/userguide/publishing_maven.html#sec:modifying_the_generated_pom\n\
-              Suppressed: DownloadException: No source artifact URL provided for 'Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0'."
+              Suppressed: DownloadException: No source artifact provided for 'Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0'."
```

Comment:
Note that adjusting the exception may break some rule violations.
The upper part of the exception says "No VCS URL provided for".

Considering the above would you agree it makes sense to not do this change?

Reply:
> Note that adjusting the exception may break some rule violations.

Don't you mean error resolutions?

> The upper part of the exception says "No VCS URL provided for".

Indeed, that should probably then also changed to "No VCS provided for" or "No VCS information provided for".

> Considering the above would you agree it makes sense to not do this change?

I admit I forgot to consider the potential impact on error resolutions. (Or really rule violations?) But I'm also not sure how real that case is. Do you have such cases?

Reply:
> Don't you mean error resolutions?

I indeed meant error resolutions.

Reply:
> Do you have such cases?

Not in the global config. I can't tell what's in the `ort.yml`s which is why I'm pointing that out.

Reply:
OK, I'll drop that change then to be on the safe side.